src/components/Layouts/Header/UnconnectedHeader.tsx

```diff
@@ -10,7 +10,9 @@ export default function UnConnectedHeader() {
         <h1>커플 연결이 필요합니다</h1>
       </S.UnConnectedTitleContainer>
       <S.SettingContainer>
-        <button>공개글 보기</button>
+        <Link to="/public/posts">
+          <button>공개글 보기</button>
+        </Link>
```

Comment:
```suggestion
        <Link to="/public/posts">공개글 보기</Link>
```
`Link` 태그 안에서 `button` 태그를 따로 둘 필요 없이 작성 가능할 것 같은데 어떻게 생각하시나요??
조금은 다르지만 저 또한 https://github.com/modern-agile-team/8term-coko-Front/pull/180#discussion_r1969674398 해당 리뷰에서 비슷하게 달린 게 있어서 말씀드려봅니다.

Reply:
@bluetree7878 리뷰 감사합니다.
제 생각에는, Link로 페이지를 이동하는 태그인 이전에, `button`이라고 생각이 되기 때문에 이렇게 작성을 했습니다.

단순히 링크를 넘나드는 요소는 많지만, 버튼을 눌러서 링크를 넘나드는걸 구분하고 싶었습니다.